projects/elife.yaml

```diff
@@ -387,6 +392,7 @@ medium:
         clustered:
             ec2:
                 cluster-size: 2
+            elb: True
```

Comment:
if the default for clusters of machines is for them to be behind an elb, this `elb: True` should be in `defaults` at the top.